core/templates/pages/volunteer-page/volunteer-page.component.html

```diff
@@ -0,0 +1,431 @@
+<div class="volunteer-page protractor-test-volunteer-page">
```

Comment:
Do we want this page only in english or do we want to provide tranlations too?

Reply:
This page is just a placeholder atm and needs to be ported onto oppia.org so that we can deprecate the foundation website ASAP. These pages will be revamped soon so we can add translations for the final design directly. Does that sg?

Reply:
sg!